### PR TITLE
feature/PLAYER-3794 ISO-639-2 instead of ISO-639-1

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
**IMPORTANT!**
Before merging the PR, [PR](https://github.com/ooyala/html5-common/pull/80) to html5-common should be merged.

We have an [issue](https://jira.corp.ooyala.com/browse/PLAYER-3794). 
The issue happened because of we use ISO-639-1, and the standard does not have special codes.
In the PR we use codes for ISO-639-2 as main codes. And the map with languages was incresed according the table from [wiki](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) .
Some of function were changed. Now we check if the code is one of special codes, and we show a label of a track according by [rules for the task](https://jira.corp.ooyala.com/browse/PLAYER-3134)

A concomitant PR is https://git.corp.ooyala.com/projects/PBW/repos/mjolnir/pull-requests/503

Unit tests are passed.

### CODE REVIEW

File: js/components/closed-caption-multi-audio-menu/helpers.js [hunk 1]
```diff
@@ -41,9 +42,8 @@ function getDisplayLanguage(languagesList, languageCode) {
   ) {
     var matchingLanguage = _.find(languagesList, function(language) {
       return (
-        language['1'] === languageCode ||
-        language['2'] === languageCode ||
         language['2T'] === languageCode ||
+        language['1'] === languageCode ||
```
Comment (aeng7) [inline js/components/closed-caption-multi-audio-menu/helpers.js]: Just to double check, do we no longer care about language['2']?

File: js/components/closed-caption-multi-audio-menu/helpers.js [hunk 2]
```diff
@@ -57,31 +57,72 @@ function getDisplayLanguage(languagesList, languageCode) {
...
+  );
+  localizedLanguage = localizedLanguage ?
+    localizedLanguage
+    :
+    phrase ? phrase : '';
+  return localizedLanguage;
+}
+
+/**
+ *
```
Comment (aeng7) [inline js/components/closed-caption-multi-audio-menu/helpers.js]: Can we add a description for this function?

**Comment by aeng7** [comment]
LGTM outside of a few minor comments/questions

File: js/components/closed-caption-multi-audio-menu/helpers.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply js/components/closed-caption-multi-audio-menu/helpers.js]: \+

File: js/components/closed-caption-multi-audio-menu/helpers.js (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply js/components/closed-caption-multi-audio-menu/helpers.js]: I found a table on [wiki](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) . It looks like the most full table with codes. I have grabbered it. So now in our map (language.js file) we have all single values for ISO-639-2 from first column and values with asterisk from first column , they are marked as "2B", and values without asterisk are marked as "2T".

About language['x']:
There is keys "1", "2B", "2T", "3" in languages.js . These are types of ISO-639 .
You can read about the types here: 
https://en.wikipedia.org/wiki/ISO_639-1
https://en.wikipedia.org/wiki/ISO_639-2
https://en.wikipedia.org/wiki/ISO_639-3

And we check if the langCode is one of these codes.

Comment (aeng7) [reply js/components/closed-caption-multi-audio-menu/helpers.js]: Thanks for the great explanation @KseniiaSokolovaEpam !

**Comment by aeng7** [comment]
Note there is a conflict in the mjolnir PR

**Comment by pilievOoyala** [comment]
LGTM but please remove that debug log file first and then you can merge this yourself.

**Comment by KseniiaSokolovaEpam** [comment]
@pilievOoyala ok (I deleted unnecessary Stereoscopic, Video and npm-debug.log files), thank you!

**Pull request merged**